### DISCUSSION TITLE
[Problem/Question]:  How can I induct frame failure render error?

### DISCUSSION THREAD

**Discussion by omonimus1** [MicrosoftEdge/WebView2Feedback#4402] [Q&A]
### What happened?

HI Webview2 team, I am testing how my app would react to webview2 crash handling.
I can use edge://kill, edge://crash, to induct crashes, but I want to specifically collect informations associated to a failed frame rendered, once failure type is == to COREWEBVIEW2_PROCESS_FAILED_KIND_FRAME_RENDER_PROCESS_EXITED

How can I programmatically create an utility, to create such crash type?

### Importance

Important. My app's user experience is significantly compromised.

### Runtime Channel

Stable release (WebView2 Runtime)

### Runtime Version

121

### Framework

----Please select----

### Operating System

Windows 11

### Repro steps

N/A

### Repros in Edge Browser

No, issue does not reproduce in the corresponding Edge version

### Regression

No, this never worked

**Comment by victorhuangwq** [comment]
I don't think that's quite possible at this point in time? @vbryh-msft  do you know?

**Comment by vbryh-msft** [comment]
I believe that `edge://crash` should crash renderer - which process failed event you are getting there? Can you create out-of-process iframe and navigate it to `edge://crash`?
